### PR TITLE
Development into main

### PR DESCRIPTION
**Pull request by HeWhoThreadsLightly**
# Pull Request

## @:ing someone
@GabrielAdward 

## Task list 
- [x] Checked the code structure so it follows guide lines
- [x] Checked that all the points in the Issue are solved
- [x] Checked that all the old tests still got approved

## Added tests
- [x] Yes, I have added tests
- [ ] No, I have not added tests
- [ ] No, I need help with adding tests

### CODE REVIEW

**Comment by GabrielAdward** [comment]
To the issue that has been solved add https://github.com/AlexTelon/FlexiCharge-My-pages/issues/158.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "@types/react-dom": "^17.0.0",
         "@types/react-router-dom": "^5.1.9",
         "axios": "^0.21.4",
-        "create-react-app": "^5.0.1",
+        "create-react-app": "^1.2.1",
```
Comment (GabrielAdward) [inline package.json:23]: Should it be downgraded to 1.2.1

**Comment by GabrielAdward** [comment]
Otherwise, it looks good

File: package.json (hunk 1 above)
Comment (HeWhoThreadsLightly) [reply package.json:23]: The downgrade happened in your PR [the bug is solved](https://github.com/AlexTelon/FlexiCharge-My-pages/commit/8fa931f47e84edc3d528296385d25d90a5dc0710)

Comment (HeWhoThreadsLightly) [reply package.json:23]: I think "create-react-app": "^5.0.1" would be the correct version. but it works right now so...

Comment (HeWhoThreadsLightly) [reply package.json:23]: I think we shuld revert the change

Comment (HeWhoThreadsLightly) [reply package.json:23]: Tests are passing localy but not on gitub, cant quite figure out what has broken stuff,
I have gotten two diffrent errors on diffrent test runs without changing the code.

    FAIL src/test/logout.test.tsx
      ● Login, check data, logout and check if information is gone

    TypeError: Cannot read properties of null (reading '_origin')

      146 |     axios.get(API_URL + "user-information", config)
      147 |       .then((response: any) => {
    > 148 |         localStorage.setItem("userProfile", JSON.stringify(response.data))
          |         ^
      149 |       });
      150 |   }
      151 |

      at Window.localStorage (node_modules/jsdom/lib/jsdom/browser/Window.js:4[20](https://github.com/AlexTelon/FlexiCharge-My-pages/actions/runs/6417721569/job/17457026140#step:5:21):50)
      at src/components/AuthService.tsx:148:9

    Test Suites: 1 failed, 2 passed, 3 total

And 

    PASS src/test/logout.test.tsx

    Test Suites: 3 passed, 3 total
    Tests:       10 passed, 10 total
    Snapshots:   0 total
    Time:        9.945 s
    Ran all test suites.
    /home/runner/work/FlexiCharge-My-pages/FlexiCharge-My-pages/node_modules/jsdom/lib/jsdom/browser/Window.js:420
          if (idlUtils.implForWrapper(this._document)._origin === "null") {
                                                 ^

    TypeError: Cannot read properties of null (reading '_origin')
    at Window.get localStorage [as localStorage] (/home/runner/work/FlexiCharge-My-pages/FlexiCharge-My-pages/node_modules/jsdom/lib/jsdom/browser/Window.js:420:50)
    at /home/runner/work/FlexiCharge-My-pages/FlexiCharge-My-pages/src/components/AuthService.tsx:[15](https://github.com/AlexTelon/FlexiCharge-My-pages/actions/runs/6417721569/job/17456038497#step:5:16)9:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    Error: Process completed with exit code 1.

**Pull request merged**